server/src/main/java/org/candlepin/model/Product.java

```diff
@@ -77,10 +77,75 @@
 @Entity
 @Table(name = "cp2_products")
 public class Product extends AbstractHibernateObject implements SharedEntity, Linkable, Cloneable {
-    public static final String UEBER_PRODUCT_POSTFIX = "_ueber_product";
+    /**
+     * Commonly used/recognized product attributes
+     */
+    public static final class Attributes {
+        /** Attribute to specify the architecture on which a given product can be installed/run */
+        public static final String ARCHITECTURE = "arch";
```

Comment:
Might be worth noting that a value of "ALL" can be used to specify all archs? Unless we want to put that in the docs instead.